lib/house.rb

```diff
@@ -0,0 +1,32 @@
+class House
+  attr_reader :price, :address, :rooms, :house_area
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @house_area = 0
+  end
+
+  def details
+    {"price" => @price.tr('$', '').to_i, "address" => @address}
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def area
```

Comment:
The logic here is great, but you don't really need an instance variable for `@house_area`; you could format the method like this:

```ruby
def area
  house_area = 0
  @rooms.each do |room|
    house_area += room.area
  end
  house_area
end
```